rag_experiment_accelerator/run/index.py

```diff
@@ -19,10 +29,23 @@
 
 def run(config_dir: str, data_dir: str = "data", filename: str = "config.json") -> None:
     """
-    Runs the main experiment loop, which chunks and uploads data to Azure AI Search indexes based on the configuration specified in the Config class.
+    Runs the indexing process based on the provided configuration.
 
-    Returns:
-        None
+    This function initializes the configuration, logs the configuration
+    details, and sets up the preprocessor.
+    It then creates the necessary directories
+    and initializes an empty index dictionary.
+    The function then creates search indexes based on the configuration.
+    After creating the indexes, it writes the index names to a file.
+    The function then splits the documents into chunks,
+    generates titles and summaries for the chunks, embeds the chunks,
+    and uploads them to Azure Search.
+
+    Args:
+        config_dir (str): The directory where the configuration file
```

Comment:
Sorry....just spotted that these args are wrong

Reply:
Fixed. thanks!